velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -15,6 +15,9 @@
  */
 #pragma once
 
+#define XXH_INLINE_ALL
+#include <xxhash.h>
```

Comment:
We can consider create a shim header file if we have to do this every time we include this